pyproject.toml

```diff
@@ -84,3 +84,10 @@ fallback_version = "0.0.0-dev"
 [tool.ruff]
 exclude = ["*_pb2.py"]
 line-length = 100
+
+select = [
+    "D", # Docstring rules
+]
+
+[tool.ruff.pydocstyle]
```

Comment:
Proposal to add Ruff docstring rules. This might be annoying at first, but it can help us track down classes and functions not commented.